### PR TITLE
KK-295 | Add option to add image to events

### PR DESCRIPTION
**Pull request by badrange**
* Switch from apollo-boost to apollo-client
* Use latest dependencies
* Upgrade Typescript too
* Disable caching for apollo client

Known issue: Right after clicking save, the image doesn't show before it is refetched from the backend.

### CODE REVIEW

File: src/domain/events/create/EventCreate.tsx [hunk 1]
```diff
@@ -32,6 +34,14 @@ const EventCreate = (props: any) => {
           selectedLanguage={selectedLanguage}
           onSelect={selectLanguage}
         />
+        <ImageInput
+          source="image"
+          label="Event image"
```
Comment (tuomas777) [inline src/domain/events/create/EventCreate.tsx:39]: Should be translatable.

File: src/domain/events/create/EventCreate.tsx [hunk 2]
```diff
@@ -32,6 +34,14 @@ const EventCreate = (props: any) => {
...
+          accept="image/*"
+          placeholder={<p>Drop your file here</p>}
```
Comment (tuomas777) [inline src/domain/events/create/EventCreate.tsx:41]: Should be translatable.

File: src/domain/events/detail/EventShow.tsx [hunk 3]
```diff
@@ -141,6 +141,11 @@ const EventShow: FunctionComponent = (props: any) => {
         <Tab label={translate('events.show.tab.label')}>
           <LanguageTabs selectedLanguage={language} onSelect={selectLanguage} />
           <ImageField source="image" />
+          <TextField
+            source={`translations.${language}.imageAltText`}
+            label={translate('events.fields.imageAltText.label')}
```
Comment (tuomas777) [inline src/domain/events/detail/EventShow.tsx:146]: For React Admin component labels, you can do this just like `label="events.fields.imageAltText.label"` (yes, Kukkuu admin does not use that everywhere ATM 🙂  )

File: src/domain/events/edit/EventEdit.tsx [hunk 4]
```diff
@@ -25,13 +27,40 @@ const EventEdit = (props: any) => {
...
+          accept="image/*"
+          placeholder={<p>Drop your file here</p>}
+        >
+          <ImageField source="image" />
+        </ImageInput>
+        <TextInput
+          source={`${translation}.imageAltText`}
+          label={translate('events.fields.imageAltText.label')}
+          validate={null}
+        />
```
Comment (tuomas777) [inline src/domain/events/edit/EventEdit.tsx:62]: The same translation-related comments apply here 🙂

**Comment by badrange** [comment]
I will resolve the comments in the next pr

File: src/domain/events/create/EventCreate.tsx (hunk 1 above)
Comment (badrange) [reply src/domain/events/create/EventCreate.tsx:39]: Fixed in #31

File: src/domain/events/create/EventCreate.tsx (hunk 2 above)
Comment (badrange) [reply src/domain/events/create/EventCreate.tsx]: Fixed in #32

File: src/domain/events/detail/EventShow.tsx (hunk 3 above)
Comment (badrange) [reply src/domain/events/detail/EventShow.tsx]: Fixed in #32

File: src/domain/events/edit/EventEdit.tsx (hunk 4 above)
Comment (badrange) [reply src/domain/events/edit/EventEdit.tsx:68]: Fixed in #32

**Pull request merged**